recipes/libfabric/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class LibfabricConan(ConanFile):
+    name = "libfabric"
+    description = "Open Fabric Interfaces"
+    topics = ("conan", "fabric")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://libfabric.org"
+    license = "BSD"
```

Comment:
```suggestion
    license = "BSD-2-Clause", "GPL-2.0-or-later"
```

https://github.com/ofiwg/libfabric/blob/v1.12.1/COPYING